### PR TITLE
[wip] [forms] no nil response on schedule-forms handler

### PR DESCRIPTION
**Pull request by teaforthecat**
I'm still going to try to fix the test, but can't get to it today.

### CODE REVIEW

File: src/election_mail_works/handlers/forms.clj [hunk 1]
```diff
@@ -72,8 +72,14 @@
...
      (when (:subscribe message)
        (h/create-and-broadcast-subscription user-id))
-     (when (and (h/user-can-get-mailing? user)
-                (seq (vals files)))
+     (cond
+       (not (h/user-can-get-mailing? user))
+       (throw (IllegalArgumentException. "User can not get mailings because user is not subscribed"))
+
+       (not (seq (vals files)))
+       (throw (IllegalArgumentException. "found no forms for given URIs"))
```
Comment (mrichards42) [inline src/election_mail_works/handlers/forms.clj]: I assume throwing here is going to return a response like `{:status :error ...}` once `handle-errors` catches it, right? What does turbovote-web do with those errors?

Comment (teaforthecat) [reply src/election_mail_works/handlers/forms.clj]: That is correct - for this case, it simply prints one message or another (pretty lame)

**Comment by mrichards42** [comment]
LGTM once that test is fixed

**Comment by teaforthecat** [comment]
the test can wait I think we should merge this - the test wasn't even working anyway

**Pull request merged**